rts/lra/run.sh

```diff
@@ -97,24 +93,42 @@ echo "Narayana installed location = $NARAYANA_INSTALL_LOCATION"
 echo "Waiting for all the servers to start"
 sleep `timeout_adjust 30 2>/dev/null || echo 30`
 
-mvn -f trip-client/pom.xml exec:java -Dexec.args="${IP_OPTS} confirm"
-mvn -f trip-client/pom.xml exec:java -Dexec.args="${IP_OPTS} cancel"
+mvn -f trip-client/pom.xml exec:java ${IP_OPTS} -Dexec.args="confirm"
```

Comment:
Considering this change to IP_OPTS, I have a feeling that you need to use MAVEN_OPTS to pass in VM options? https://www.mojohaus.org/exec-maven-plugin/usage.html

Reply:
Thanks Tom, updated: I added the line 'export MAVEN_OPTS=${IP_OPTS}' and removed the '${IP_OPTS}' from 'mvn' execution.

Reply:
@tomjenkinson could we resolve this review?

Reply:
Thank you, I will resolve it